spotirec.py

```diff
@@ -450,12 +453,12 @@ def get_device(device_name: str):
                 rec.playback_device = devices[device_name]
                 return
             except KeyError:
-                print(f'Error: device \"{device_name}\" not recognized. Saved devices:')
+                logging.error(f'Error: device \"{device_name}\" not recognized. Saved devices:')
                 for x in devices:
-                    print(x)
+                    logging.error(x)
```

Comment:
'Error' in string on 352 should be removed. Also 458 are simple info messages to print which devices are available - not sure how to handle that. Maybe those should just be removed and line 352 reformatted.